Amplify/DevMenu/View/IssueReporter.swift

```diff
@@ -112,24 +116,25 @@ struct IssueReporter: View {
 
     /// Copy issue as a markdown string to clipboard
     private func copyToClipboard() {
-        UIPasteboard.general.string =
-            IssueInfoHelper.generateMarkdownForIssue(
-                issue: IssueInfo(issueDescription: issueDescription,
-                                 includeEnvInfo: includeEnvInfo,
-                                 includeDeviceInfo: includeDeviceInfo)
-            )
+        let issue = IssueInfo(issueDescription: issueDescription,
+                              includeEnvInfo: includeEnvInfo,
+                              includeDeviceInfo: includeDeviceInfo)
+        let value = IssueInfoHelper.generateMarkdownForIssue(issue: issue)
+#if canImport(UIKit)
+        UIPasteboard.general.string = value
+#elseif canImport(AppKit)
```

Comment:
This would never get hit considering the entire file is wrapped in an `#if canImport(UIKit)`, right?